src/components/Tooltip/Tooltip.tsx

```diff
@@ -561,28 +562,34 @@ const Tooltip = ({
       role="tooltip"
       className={classNames(
         'react-tooltip',
-        styles['tooltip'],
+        coreStyles['tooltip'],
         styles[variant],
         className,
         `react-tooltip__place-${actualPlacement}`,
         {
-          [styles['show']]: canShow,
-          [styles['fixed']]: positionStrategy === 'fixed',
-          [styles['clickable']]: clickable,
+          [coreStyles['show']]: canShow,
+          [coreStyles['fixed']]: positionStrategy === 'fixed',
+          [coreStyles['clickable']]: clickable,
         },
       )}
       style={{ ...externalStyles, ...inlineStyles }}
       ref={tooltipRef}
     >
       {content}
       <WrapperElement
-        className={classNames('react-tooltip-arrow', styles['arrow'], classNameArrow, {
-          /**
-           * changed from dash `no-arrow` to camelcase because of:
-           * https://github.com/indooorsman/esbuild-css-modules-plugin/issues/42
-           */
-          [styles['noArrow']]: noArrow,
-        })}
+        className={classNames(
+          'react-tooltip-arrow',
+          coreStyles['arrow'],
+          styles['arrow'],
```

Comment:
Is keeping both styles intentional?

Reply:
yes, but please feel free to reorganize the styles, I tried to keep the necessary CSS for position and basic behavior into the Core file, I'll check again this week when I have more time, but please, go ahead if you want to change anything